blueoil/datasets/ilsvrc_2012.py

```diff
@@ -72,17 +71,14 @@ def _init_files_and_annotations(self):
     def _files_and_annotations(self):
         txt_file = self.texts[self.subset]
 
-        df = pd.read_csv(
-            txt_file,
-            delim_whitespace=True,
-            header=None,
-            names=['filename', 'class_id'])
+        files, labels = list(), list()
+        with open(txt_file) as f:
```

Comment:
Python `csv` module can load space separated file.
https://docs.python.org/ja/3/library/csv.html#csv.Dialect.delimiter

It's good to use standard library than implement by ourself.

Reply:
@iizukak 
If I implement this block using csv, the code will be like the followed:
```
import csv

files, labels = list(), list()
with open(txt_file) as f:
    reader = csv.reader(f, delimiter=' ')
    for lines in reader:
        files.append(lines[0])
        labels.append(lines[1])
```

In term of amount of code, both are about the same, except csv requires extra `import csv` line, and this also means an extra library is required in this code.
Since there is no advantage in simplicity, is it still better to use csv library in this case?
I don't have much knowledge in good programming style, so I hope to get my doubt cleared in this discussion.
Thank you!